macos/library/FLEViewController.m

```diff
@@ -383,25 +393,50 @@ - (void)initPlugins {
 
   FLETextInputPlugin *textPlugin = [[FLETextInputPlugin alloc] init];
   [self _addPlugin:textPlugin];
+    
+  FLEKeyEventPlugin *keyEventPlugin = [[FLEKeyEventPlugin alloc] init];
+  [self _addPlugin:keyEventPlugin];
+  [self.additionalKeyResponders addObject:keyEventPlugin];
 }
 
+
 - (BOOL)acceptsFirstResponder {
-  return YES;
+    return YES;
 }
 
+#pragma mark - NSResponder key events
+
+- (void)keyDown:(NSEvent *)event {
+    for (NSResponder* responder in self.additionalKeyResponders) {
```

Comment:
This seems to work as is, when using text input and key events the order is as follows:
* KeyDown
* Text
* KeyUp

Because the keyevent plugin is added first it handles the event first. I changed the NSMutableSet to NSOrderedMutableSet to guarantee this.